rosdep/python.yaml

```
@@ -7606,6 +7606,10 @@ python3-pil:
     slackpkg:
       packages: [python3-pillow]
   ubuntu: [python3-pil]
+python3-pil-imagetk:
+  debian: [python3-pil.imagetk]
+  fedora: [python3-pillow-tk]
```

Would you mind adding RHEL as well?
RHEL 8: https://pkgs.org/search/?q=python3-pillow-tk
RHEL 9: https://packages.fedoraproject.org/pkgs/python-pillow/python3-pillow-tk/ (via EPEL)

```suggestion
  fedora: [python3-pillow-tk]
  rhel: [python3-pillow-tk]
```

---

Sure, I missed them. Thank you.